### ISSUE TITLE
Samples build error on CJK Windows 

### ISSUE THREAD

**Issue by hwan-ajou** [LunarG/VulkanSamples#240]
When I build samples on my Windows 10 (Korean) - There is error related to language locale setting. (message is about unicode .... etc) I solved this problem with change locale to support unicode. Easy way to solve this problem is change bat file like this. 

echo off
:: set language locale to Unicode
chcp 65001 

:: Check that cmake is configured
where cmake.exe > nul 2>&1
if not %errorlevel% equ 0 (
    echo ERROR: CMake was not found. Please install CMake or put cmake.exe in your PATH.
    exit /b 1
)

**Comment by Tony-LunarG** [comment]
Could you please post the build output with the error message that you are seeing?  And what is happening when you get the error?

**Comment by hwan-ajou** [comment]
Errors Look like this (Repeated)
```
[D:\VulkanSDK\1.1.106.0\glslang\build\External\spirv-tools\source\opt\SPIRV-Tools-opt.vcxproj]
cl : 명령줄 warning D9025: '/W3'을(를) '/w'(으)로 재정의합니다. [D:\VulkanSDK\1.1.106.0\glslang\build\External\spirv-tools\source\opt\SPIRV-Tools-opt.vcxproj]
D:\VulkanSDK\1.1.106.0\glslang\glslang\MachineIndependent\iomapper.cpp : warning C4819: 현재 코드 페이지(949)에서 표시할 수 없는 문자가 파일에 들어 있습니다. 데이터가 손실되지 않게 하려면 해당 파일을 유니코드 형식으로 저장하십시오. [D:\VulkanSDK\1.1.106.0\glslang\build\glslang\glslang.vcxproj]
```
It said "Current Code Page(949) there is character set included which is not possible to present...The file should be saved with unicode...."
It is because normal windows 10 Korean setting is code set as 949
check with $ chcp command, then it will say 949. 
To solve this problem before run $ build_windows_samples.bat , run $ chcp 65001
It changes language code set with unicode. You can build without error.

**Comment by hwan-ajou** [comment]
I think that if build_windows_samples.bat include chcp 65001 command after echo off, it will solve all issues.

**Comment by hwan-ajou** [comment]
I think that if build_windows_samples.bat include chcp 65001 after echo off, it will solve the issue.
When failed build with code set 949, it is not possible to rebuild with chcp 65001. 
We should remove SDK and reinstall SDK, then chcp 65001 and rebuild will work well.

**Comment by Tony-LunarG** [comment]
Fixed with #242.  Thanks!
